src/Sarif.Multitool/FileWorkItemsOptions.cs

```diff
@@ -9,10 +10,40 @@ namespace Microsoft.CodeAnalysis.Sarif.Multitool
     public class FileWorkItemsOptions
     {
         [Option(
-            'f',
+            't',
             "filing-target-type",
             HelpText = "The type of system to which work items are to be filed. Must be one of 'github' and 'azureDevOps'.",
             Required = true)]
-        public string TargetType { get; internal set; }
+        public string FilingTargetType { get; internal set; }
+
+        [Option(
+            'p',
+            "project-uri",
+            HelpText = "The absolute URI of the project under which the work items are to be filed.",
+            Required = true)]
+        public string ProjectUriString { get; internal set; }
+
+        // For an explanation of the difference between ProjectUriString and ProjectUri, see FileWorkItemsCommand.ValidateOptions.
+        public Uri ProjectUri { get; internal set; }
+
+        [Option(
+            'f',
+            "filtering-strategy",
+            HelpText = "Specifies the strategy for selecting which results to file. Must be one of 'new' and 'all'.",
+            Required = true)]
```

Comment:
>Required = true [](start = 12, length = 15)

We should have reasonable defaults. See the other PR for the behavior I suggest (either pull in all results, if no baseline data, otherwise, process updated and new).

Reply:
Will do later. Not as important as demonstrating bug filing this week.

---
In reply to: [306575068](https://github.com/microsoft/sarif-sdk/pull/1583#discussion_r306575068) [](ancestors = 306575068)